src/main/java/cz/it4i/fiji/datastore/register_service/DatasetRegisterServiceEndpoint.java

```diff
@@ -152,26 +153,59 @@ public Response startDatasetServer(@PathParam(UUID) String uuid,
 		}
 	}
 
+	@GET
+	@Path("datasets/{" + UUID + "}/type")
+	public Response getDatasetType(@PathParam(UUID) String uuid)
+	{
+		fileTypeEnum type=datasetRegisterServiceImpl.getDataType(uuid);
+
+		return Response.ok().entity(type.toString()).type(
+				MediaType.TEXT_PLAIN).build();
+
+	}
+
+
 	@POST
 	@Path("datasets/")
 	@Consumes(MediaType.APPLICATION_JSON)
 	public Response createEmptyDataset(DatasetDTO dataset)
 	{
+		String test=dataset.getCompression();
```

Comment:
I suggest to NOT pass type in compression. I guess that more intuitive is attribute type in an object of DatasetDTO class.